pandas/tests/frame/test_alter_axes.py

```diff
@@ -201,7 +201,8 @@ def test_set_index_pass_arrays_duplicate(self, frame_of_index_cols, drop,
         # need to adapt first drop for case that both keys are 'A' --
         # cannot drop the same column twice;
         # use "is" because == would give ambiguous Boolean error for containers
-        first_drop = False if (keys[0] is 'A' and keys[1] is 'A') else drop
+        first_drop = False if (
+            keys[0] is 'A' and keys[1] is 'A') else drop  # noqa: F632
```

Comment:
F632 is ignored because it calls out using `is` for equality comparisons. You can see in the note above though that this test intentionally uses `is` instead of the equality operator

Reply:
IIRC interning of short strings is a CPython implementation detail.  Are we sure we want to rely on that?